### PR TITLE
Restore returning the subscription expire time instead of bool

### PR DESCRIPTION
**Pull request by benthecarman**
small fix so we can display when the next mutiny+ invoice is

### CODE REVIEW

**Review by TonyGiorgio — APPROVED** [review]

File: mutiny-wasm/src/lib.rs [hunk 1]
```diff
@@ -262,7 +261,11 @@ impl MutinyWallet {
         let inner =
             mutiny_core::MutinyWallet::new(storage, config, Some(logger.session_id.clone()))
                 .await?;
-        Ok(MutinyWallet { mnemonic, inner })
+        Ok(MutinyWallet {
+            mnemonic,
+            inner,
+            subscription_expire_time: sub_expired_time,
```
Comment (TonyGiorgio) [inline mutiny-wasm/src/lib.rs:267]: really? that's such a hack

Comment (benthecarman) [reply mutiny-wasm/src/lib.rs:267]: not sure a better place to put it, maybe the config?

Comment (TonyGiorgio) [reply mutiny-wasm/src/lib.rs:267]: Honestly I was fine with the old way of doing it. Now I'm questioning if you've covered every edge case paul was doing in the frontend previously.

Comment (TonyGiorgio) [reply mutiny-wasm/src/lib.rs:267]: Where do you update this when the user subscribes?

Comment (TonyGiorgio) [reply mutiny-wasm/src/lib.rs:267]: Honestly I would rather revert the subscription changes at this point.

**Pull request closed without merging**